### PR TITLE
T1552   unsecured credentials   unprivileged process opened a registry hive

### PR DESCRIPTION
**Pull request by TalNos**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-10938

## Description
New playbook for - T1552 - Unsecured Credentials - Unprivileged process opened a registry hive.

### CODE REVIEW

File: Packs/Core/Playbooks/playbook-Unprivileged_process_opened_a_registry_hive.yml [hunk 1]
```diff
@@ -0,0 +1,563 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "61":
+    id: "61"
+    taskid: 937d2db4-abbb-41d0-8385-dacbd145654c
+    type: condition
+    task:
+      id: 937d2db4-abbb-41d0-8385-dacbd145654c
+      version: -1
+      name: Check the signature of the Actor & CGO process and the prevalence of the Actor & CGO process and Actor & CGO CommandLine
```
Comment (altmannyarden) [inline Packs/Core/Playbooks/playbook-Unprivileged_process_opened_a_registry_hive.yml:457]: As a user, I dont understand anything from this name. 
Please change it. 
Check the signature, what are you checking?

File: Packs/Core/Playbooks/playbook-Unprivileged_process_opened_a_registry_hive.yml [hunk 2]
```diff
@@ -0,0 +1,563 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "53":
+    id: "53"
+    taskid: 76b1f4c5-07a8-45d9-8f47-a2e6e65215cc
+    type: condition
+    task:
+      id: 76b1f4c5-07a8-45d9-8f47-a2e6e65215cc
+      version: -1
+      name: Found any related alerts?
```
Comment (altmannyarden) [inline Packs/Core/Playbooks/playbook-Unprivileged_process_opened_a_registry_hive.yml:373]: change to Found related alerts requiring causality termination?

File: Packs/Core/ReleaseNotes/3_0_82.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+- This Playbook is designed to handle the 'Unprivileged process opened a registry hive' alerts and executes the following:
+
+Investigation:
+Check the signature of the Actor & CGO processes. 
+Check the prevalence of the Actor & CGO process and Actor & CGO CommandLine
+Check for related XDR alerts using MITRE tactics to identify any malicious activity.
+
+Remediation:
+Based on the signature status of the Actor & CGO processes, prevalence of the Actor & CGO process and Actor & CGO CommandLine, and related alerts if found, the playbook will terminate the causality process if any malicious parameters are found. (Available from Cortex XSOAR 6.10.0).
```
Comment (altmannyarden) [inline Packs/Core/ReleaseNotes/3_0_82.md:14]: The remediation part is like the condition task name that I dont like (I think its unclear) please rephrase this sentence  (also in the playbook description)

**Comment by altmannyarden** [comment]
@TalNos Trigger is missing

File: Packs/Core/Playbooks/playbook-Unprivileged_process_opened_a_registry_hive.yml (hunk 2 above)
Comment (TalNos) [reply Packs/Core/Playbooks/playbook-Unprivileged_process_opened_a_registry_hive.yml:373]: done

File: Packs/Core/Playbooks/playbook-Unprivileged_process_opened_a_registry_hive.yml (hunk 1 above)
Comment (TalNos) [reply Packs/Core/Playbooks/playbook-Unprivileged_process_opened_a_registry_hive.yml:457]: changed to 'Check for process signatures and prevalence'.

File: Packs/Core/Playbooks/playbook-Unprivileged_process_opened_a_registry_hive.yml [hunk 4]
```diff
@@ -0,0 +1,563 @@
+id: Unprivileged process opened a registry hive
+version: -1
+name: Unprivileged process opened a registry hive
+description: "This Playbook is designed to handle the 'Unprivileged process opened a registry hive' alerts and executes the following:\n\nInvestigation:\nCheck the signature of the Actor & CGO processes. \nCheck the prevalence of the Actor & CGO process and Actor & CGO CommandLine\nCheck for related XDR alerts using MITRE tactics to identify any malicious activity.\n\nRemediation:\nBased on the signature status of the Actor & CGO processes, prevalence of the Actor & CGO process and Actor & CGO CommandLine, and related alerts if found, the playbook will terminate the causality process if any malicious parameters are found."
+tags:
+- TA0006 - Credential Access
```
[the remark this answers is not in the document]
Comment (TalNos) [reply Packs/Core/Playbooks/playbook-Unprivileged_process_opened_a_registry_hive.yml:6]: added the 'T1552 - Unsecured Credentials' tag

File: Packs/Core/Playbooks/playbook-Unprivileged_process_opened_a_registry_hive.yml [hunk 5]
```diff
@@ -0,0 +1,572 @@
+id: Unprivileged process opened a registry hive
+version: -1
+name: Unprivileged process opened a registry hive
+description: "This Playbook is designed to handle the 'Unprivileged process opened a registry hive' alerts and execute the following:\n- Check for process signatures and prevalence.\n- Searching for additional Cortex XSIAM suspicious alerts in the incident is conducted by searching relevant Mitre techniques.\n\nRemediation:\n- Handles malicious alerts by terminating causality processes."
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Unprivileged_process_opened_a_registry_hive.yml:4]:
```suggestion
description: "This playbook is designed to handle the 'Unprivileged process opened a registry hive' alerts and execute the following:\n- Check for process signatures and prevalence.\n- Searching for additional Cortex XSIAM suspicious alerts in the incident is conducted by searching relevant Mitre techniques.\n\nRemediation:\n- Handles malicious alerts by terminating causality processes."
```

File: Packs/Core/Playbooks/playbook-Unprivileged_process_opened_a_registry_hive.yml [hunk 6]
```diff
@@ -0,0 +1,572 @@
...
+    isautoswitchedtoquietmode: false
+  "44":
+    id: "44"
+    taskid: 041c6225-6062-47ad-86be-3b7d81f4fb19
+    type: regular
+    task:
+      id: 041c6225-6062-47ad-86be-3b7d81f4fb19
+      version: -1
+      name: Terminate Causality (CGO)
+      description: Terminate a process tree by its causality ID. Available only for XSIAM 2.4.
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Unprivileged_process_opened_a_registry_hive.yml:183]:
```suggestion
      description: Terminate a process tree by its causality ID. Available only for Cortex XSIAM 2.4.
```

File: Packs/Core/Playbooks/playbook-Unprivileged_process_opened_a_registry_hive_README.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
+This Playbook is designed to handle the 'Unprivileged process opened a registry hive' alerts and execute the following:
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Unprivileged_process_opened_a_registry_hive_README.md:1]:
```suggestion
This playbook is designed to handle the 'Unprivileged process opened a registry hive' alerts and execute the following:
```

File: Packs/Core/Playbooks/playbook-Unprivileged_process_opened_a_registry_hive_README.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+This playbook does not use any sub-playbooks.
+
+### Integrations
+
+* CortexCoreIR
+
+### Scripts
+
+* SearchIncidentsV2
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Unprivileged_process_opened_a_registry_hive_README.md:22]:
```suggestion
CortexCoreIR

### Scripts

SearchIncidentsV2
```

File: Packs/Core/ReleaseNotes/3_0_82.md [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+#### Playbooks
+
+##### New: Unprivileged process opened a registry hive
+
+- This Playbook is designed to handle the 'Unprivileged process opened a registry hive' alerts and executes the following:
+
+Investigation:
+Check the signature of the Actor & CGO processes. 
+Check the prevalence of the Actor & CGO process and Actor & CGO CommandLine
+Check for related XDR alerts using MITRE tactics to identify any malicious activity.
```
Comment (ShirleyDenkberg) [inline Packs/Core/ReleaseNotes/3_0_82.md:11]:
```suggestion
Check for related Cortex XDR alerts using MITRE tactics to identify any malicious activity.
```

**Comment by ShirleyDenkberg** [comment]
@altmannyarden Doc review completed.

File: Packs/Core/Playbooks/playbook-Unprivileged_process_opened_a_registry_hive_README.md
Comment (umishkin) [inline Packs/Core/Playbooks/playbook-Unprivileged_process_opened_a_registry_hive_README.md:1]: Please change the README to the updated format 
Ask it from @idovandijk

File: Packs/Core/ReleaseNotes/3_0_82.md
Comment (umishkin) [inline Packs/Core/ReleaseNotes/3_0_82.md:1]: Same for the release notes - take it from @idovandijk 
or from this playbooks:

Comment (umishkin) [reply Packs/Core/ReleaseNotes/3_0_82.md:1]: sent a pic by slack to @TalNos

File: Packs/Core/Playbooks/playbook-Unprivileged_process_opened_a_registry_hive_README.md
Comment (TalNos) [reply Packs/Core/Playbooks/playbook-Unprivileged_process_opened_a_registry_hive_README.md:1]: done

File: Packs/Core/ReleaseNotes/3_0_82.md
Comment (TalNos) [reply Packs/Core/ReleaseNotes/3_0_82.md:1]: done

File: Packs/Core/ReleaseNotes/3_0_83.md [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+
+Playbook Stages:
+
+Investigation:
+- The playbook is designed to investigate and respond to unprivileged processes opening registry hives. It examines the unprivileged process that triggered the alert, the command line, and searches for additional suspicious Cortex XSIAM alerts within the same incident in order to determine whether a remediation measure is required.
+
+Remediation:
+- To prevent malicious activity from continuing, the playbook terminates the causality processes that triggered the alert. (Available from Cortex XSOAR 8.8.0).
+
+#### Triggers Recommendations
```
Comment (TalNos) [inline Packs/Core/ReleaseNotes/3_0_83.md:16]: @umishkin In last RN's, I didn't see the 'Triggers Recommendations' section; should we keep it or remove it?

File: Packs/Core/Triggers/Trigger_-_Unprivileged process opened a registry hive.json [hunk 11]
```diff
@@ -0,0 +1,18 @@
+{
+  "trigger_id": "a8782c70ecf48029bd6c6634f1b5beb5",
```
Comment (altmannyarden) [inline Packs/Core/Triggers/Trigger_-_Unprivileged process opened a registry hive.json:2]: fix the file name

Comment (TalNos) [reply Packs/Core/Triggers/Trigger_-_Unprivileged process opened a registry hive.json]: done

File: Packs/Core/ReleaseNotes/3_0_83.md (hunk 10 above)
Comment (TalNos) [reply Packs/Core/ReleaseNotes/3_0_83.md:16]: Uriel said we should keep the 'Triggers Recommendations' section.

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**